configure.ac

```diff
@@ -1182,6 +1188,36 @@ echo Build HQ3X scaler.......... : $enable_hq3x
 echo Build HQ4X scaler.......... : $enable_hq4x
 fi
 echo Build NxBR scalers......... : $enable_nxbr
+echo
+echo Audio :
+echo Enable ALSA midi........... : $enable_alsa
+echo Enable Fluidsynth midi..... : $enable_fluidsynth
+echo Enable mt32emu midi........ : $enable_mt32emu
+echo Enable Timidity midi....... : $enable_timidity_midi
+echo Enable SFX as midi......... : $enable_midi_sfx
+echo
+echo Addons :
+echo Create data files.......... : $enable_data
+echo Create zipped Savegames.... : $enable_zip_support
+echo Build GIMP plugin.......... : $enable_gimp_plugin
+echo Build Gnome SHP Thumbnailer : $enable_gnome_shp_thumbnailer
+echo
+echo Debug :
+echo Build Usecode debugger..... : $enable_usecode_debugger
+echo Debug Usecode Container.... : $enable_usecode_container
+echo Debug non Readied Objects.. : $enable_nonreadied_objects
+echo
+echo Compilation :
+echo Enable compiler sanitizers. : $enable_sanitizers
+echo Enable SDL parachute....... : $enable_sdl_parachute
+echo Build with static libraries : $enable_static_libs
+echo Build libexult library..... : $enable_libexult
+echo Turn all warnings to errors : $enable_warning_errors
+echo Turn pedantics to errors... : $enable_pedantic_errors
+echo
+echo enable_timidity............ : $enable_timidity
+echo enable_gtk_interface....... : $enable_gtk_interface
```

Comment:
This variable, on the other hand, was only ever set to 'no' in one spot and not used for anything, ever, as far back as we have a configure.ac. It probably can go away too.